README.md

```diff
@@ -112,129 +111,136 @@ lock_expiration = 1800
 
 To use `cs3client`, you first need to import and configure it. Here's a simple example of how to set up and start using the client. For configuration see [Configuration](#configuration). For more in depth examples see `cs3-python-client/examples/`. 
 
-### Initilization
+### Initilization and Authentication
 ```python
 import logging
 import configparser
-from cs3client import CS3Client
-from cs3resource import Resource
+from cs3client.cs3client import CS3Client
 
 config = configparser.ConfigParser()
 with open("default.conf") as fdef:
     config.read_file(fdef)
-
 log = logging.getLogger(__name__)
 
 client = CS3Client(config, "cs3client", log)
-# client.auth.set_token("<your_token_here>")
-# OR
+
+# Set client secret
 client.auth.set_client_secret("<your_client_secret_here>")
+# Checks if token is expired if not return ('x-access-token', <token>)
+# if expired, request a new token from reva
+auth_token = client.auth.get_token()
+
+# OR if you already have a reva token
+# Checks if token is expired if not return (x-access-token', <token>)
+# if expired, throws an AuthenticationException (so you can refresh your reva token)
+token = "<your_reva_token>"
+auth_token = client.auth.check_token(token)
```

Comment:
Please check my comment about removing auth from the client. Your examples should import `Auth`, create an object, and then use it.